docs/pages/becoming_a_maker/overview.mdx

```diff
@@ -0,0 +1,274 @@
+| `network` | The Bitcoin network the asb will connect to. |
+
+### Monero Section
+
+The `monero` section specifies a few details about the asb's interaction with the Monero blockchain.
+We do not recommend changing these settings, however we document them for completeness sake.
+
+```toml filename="config_mainnet.toml"
+# ...
+
+[monero]
+wallet_rpc_url = "http://mainnet_monero-wallet-rpc:18083/json_rpc"
+network = "Mainnet"
+
+# ...
+```
+
+| Option | Description |
+| --- | --- |
+| `wallet_rpc_url` | The URL of the _monero-wallet-rpc_ server the asb will connect to. The _monero-wallet-rpc_ server is is used to interact with the Monero blockchain. The default URL points to the docker-hosted _monero-wallet-rpc_ server. |
+| `network` | The Monero network the asb will connect to. |
+
+### Tor Section
+
+The `tor` section specifies the asb's onion service (hidden service) configuration.
+An onion service can be used to make the asb publicly accessible without exposing the server's IP address or requiring an opened port ([further reading](https://community.torproject.org/onion-services/overview/)).
+
+```toml filename="config_mainnet.toml"
+# ...
+
+[tor]
+register_hidden_service = true
+hidden_service_num_intro_points = 5
+
+# ...
+```
+
+| Option | Description |
+| --- | --- |
+| `register_hidden_service` | Whether the asb should register an onion service.  |
+| `hidden_service_num_intro_points` | If the asb registers an onion service, this specifies the number of introduction points the asb will use. |
+
+
+### Network Section
+
+The `network` section specifies the asb's networking configuration.
+This includes:
+ - which rendezvous points the asb will connect to,
+ - on which port, and
+ - external addresses the asb will advertise to the rendezvous points.
+
+<Callout type="info">
+These addresses are specified using the multiaddr format ([link](https://multiformats.io/multiaddr)).
+A multiaddr combines all necessary information into a single string.
+
+For example, the first rendezvous point's multiaddr specifies the rendezvous point at the DNS address `discover.unstoppableswap.net`, on port `8888` with the peer ID `12D3KooWA6cnqJpVnreBVnoro8midDL9Lpzmg8oJPoAGi7YYaamE`.
+</Callout>
+
+```toml filename="config_mainnet.toml"
+# ...
+
+[network]
+# the ip and port the asb will listen on
+listen = ["/ip4/0.0.0.0/tcp/9939"] 
+
+# the rendezvous points the asb will connect to (address, port and peer id)
+rendezvous_point = [
+    "/dns4/discover.unstoppableswap.net/tcp/8888/p2p/12D3KooWA6cnqJpVnreBVnoro8midDL9Lpzmg8oJPoAGi7YYaamE", 
+    "/dns4/eratosthen.es/tcp/7798/p2p/12D3KooWAh7EXXa2ZyegzLGdjvj1W4G3EXrTGrf6trraoT1MEobs"
+]
+
+# the external addresses the asb will advertise to the rendezvous points (only address)
+external_addresses = [
+    # e.g. "/dns4/your.domain.com"
```

Comment:
1. the protocol is missing here (tcp/9939)

I'm 70% sure that we have to include the peer id here as well in the advertised address